### PR TITLE
Min io

### PR DESCRIPTION
**Pull request by ZombieAlienRobot**
## Description of changes
Added MinIO with some simple Endpoints for up- and downloading files.

  
## How has this been tested?
Please describe the test strategy you followed.

- [ ] automated unit test
- [ ] automated integration test
- [ ] automated acceptance test
- [X] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.

Instructions:

Start the service with "docker compose up -d".
Wait until all the containers have started and the app is running.
Use GraphQL to create a new upload link. 
Copy the url link from the response. 
Make a PUT Request with REST:  add the file you want to upload to the body of the request.
Use GraphQL to generate a new download link.
Make a GET Request to download the file from the generated link.
    
## Checklist before requesting a review
- [X] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My code fulfilles all acceptance criteria
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I have made corresponding changes to the documentation
- [X] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [X] I have updated the changes in the ticket description

## Checklist for reviewer
- [x] The code works and does not throw errors
- [x] The code is easy to understand and there are no confusing parts
- [x] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [x] The code change accomplishes what it is supposed to do
- [x] I cannot think of any use case in which the code does not behave as intended
- [x] The added and existing tests reasonably cover the code change
- [x] I cannot think of any test cases, input or edge cases that should be tested in addition
- [x] Description of the change is included in the documentation

### CODE REVIEW

File: src/main/resources/graphql/schema.graphqls [hunk 1]
```diff
@@ -55,4 +72,8 @@ type Mutation {
   createMediaRecord(input: CreateMediaRecordInput!): MediaRecord!
   updateMediaRecord(input: UpdateMediaRecordInput!): MediaRecord!
   deleteMediaRecord(id: UUID!): UUID!
-}
\ No newline at end of file
+  Storage_createUploadUrl(input: InputStorage_CreateUrl): Storage_UploadUrl
```
[the remark this answers is not in the document]
Comment (ZombieAlienRobot) [reply src/main/resources/graphql/schema.graphqls:75]: Nah I just followed a guide, renamed it now

File: src/main/java/de/unistuttgart/iste/gits/media_service/service/MediaService.java
[the remark this answers is not in the document]
Comment (ZombieAlienRobot) [reply src/main/java/de/unistuttgart/iste/gits/media_service/service/MediaService.java:1]: Wrong setting in my IDE

**Review by myluki2000 — APPROVED** [review]
